addons/contexts/README.md

```diff
@@ -178,13 +178,13 @@ An array of params contains a set of predefined `props` for `components`. This o
 
 (required)
 
-An unique name for representing the props.
+A unique name for representing the props.
```

Comment:
not a grammar pro but shouldnt this stay `An` as unique starts with a vowel?

Reply:
It is a vowel, but the sound is what matters. A consonant sound, which unique makes here ("yun" with a hard y).